src/main/java/com/thealgorithms/strings/WordUpperCase.java

```diff
@@ -0,0 +1,28 @@
+package com.thealgorithms.strings;
+
+public class WordUpperCase {
```

Comment:
The class name 'WordUpperCase' could be more descriptive of its purpose.
For example, 'Capitalizer' or 'StringCapitalizer'.